services/src/services/errorTelemetry/errorTelemetry.class.js

```diff
@@ -27,9 +27,9 @@ class ErrorTelemetryService {
       return Promise.reject({status:'KO'});
     }
 
-    // FIXME: TBD where, what and how to actually send data
     const toWrite = `${new Date()} => ${JSON.stringify(data)}\n\n`;
```

Comment:
I do not think that comment should be removed yet, or the format should be evolved a bit. As it is now, it's sending a somewhat still raw: "date (without defined format) => the json of the log", which I fear is not going to be greatly parsed by Sentry is it? Anyway, I guess we should clean up that too.